builder/src/firmware.rs

```diff
@@ -222,7 +222,7 @@ pub mod driver_tests {
 
     pub const KEYVAULT_FPGA: FwId = FwId {
         bin_name: "keyvault",
-        features: &["fpga_realtime"],
+        features: &["emu", "fpga_realtime"],
```

Comment:
Should this change be checked in? I think the above `KEYVAULT` binary has the emu feature.

Reply:
I'm not sure why this works without it, tbh. If you don't include the `emu` feature, then the test harness won't generate the correct code to exit the test, so it will hang forever.

It is required for the DMA test to work, so I thought I would make the keyvault test match.